### PR TITLE
Docs Improvement

### PR DESCRIPTION
**Pull request by ExoPlant**
Improve docs

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Index
+
+## Getting Started
+* [Basic Usage](./basic-usage.md): *How to get the `packwiz-gui` running*
+* [Building](./build.py.md): *How to build `packwiz-gui`*
+* [Command Line](./command-line.md): *Command line options for `packwiz-gui`*
+* [Windows](./windows.md) *More info about Windows for `packwiz-gui`*
+
+## Bug Reporting
+* [Known Bugs](./bugs.md): *A list of known bugs that are difficult or impossible to fix by us, and workarounds for some of them.*
```
Comment (Samisafool) [inline docs/README.md:10]: Needs period (fullstop) at the end of every sentence

File: docs/windows.md [hunk 2]
```diff
@@ -4,5 +4,7 @@ Ah yes, windoze.
 
 Known windows differences:
 
-1. packwiz-gui is less tested on windows, (linux is superior) and may crash or not work properly. File a github issue.
+1. packwiz-gui is less tested on windows, (obviously linux is superior) and may crash or not work properly. File a github issue.
 2. Need to use `python` instead of `python3` on the command line if you don't use windows store python.
+
+*Note: even though Windows is less supported it doesn't mean we won't fix bugs for it ASAP*
```
Comment (Samisafool) [inline docs/windows.md:10]: Unnecessary note, this is obvious since you can sense sarcasm above

**Review by Samisafool — CHANGES_REQUESTED** [review]
e

File: docs/windows.md (hunk 2 above)
Comment (ExoPlant) [reply docs/windows.md:10]: Nah, linux superior, no sarcasm

File: docs/README.md (hunk 1 above)
Comment (ExoPlant) [reply docs/README.md:10]: bruh

File: docs/windows.md (hunk 2 above)
Comment (Samisafool) [reply docs/windows.md:10]: Ok but still obvious lol

Just because linux is superior obviously doesnt mean anything

Comment (ExoPlant) [reply docs/windows.md:10]: I mean it says it's less tested. I'm just saying it makes it clear that we do still support the Win version

Comment (Samisafool) [reply docs/windows.md:10]: You never said you didn't support it so unnecessary note

Comment (Samisafool) [reply docs/windows.md:10]: oooooor the note should be "we still support windows btw"

Comment (ExoPlant) [reply docs/windows.md:10]: Just a fancier way of saying that xD

Comment (Samisafool) [reply docs/windows.md:10]: whatever lol

**Review by Samisafool — APPROVED** [review]

**Pull request merged**